fs/src/writer_file.rs

```diff
@@ -1,12 +1,9 @@
 use super::LocalFile;
 use super::directory_manager::DirectoryManagementError;
 use super::utils::{FsError, validate_path};
+#[cfg(target_family = "unix")]
```

Comment:
Nit: should we move the import to the unix block? As we do in the other modules.